doc/RELEASE-NOTES.md

```diff
@@ -0,0 +1,190 @@
+v0.70.0 (Beta) Release Notes
+====
+This is a very *significant* "minor" release of the pbench code base, agent and server.
+
+The "Tool Meister" functionality (PR #1248) is the major piece of functionality delivered with this release.  This is a significant change, where the pbench-agent first orchestrates the instantiation of a "Tool Meister" process on all hosts registered with tools, using a Redis instance to coordinate their operation, and the new "Tool Data Sink" handles the collection of data into the pbench run directory hierarchy.  We effectively eliminate all remote SSH operations for tools except one per host to orchestrate the creation of the Tool Meister instance.
+
+Along with the delivery of the "Tool Meister" work, the notion of tool registration has changed significantly, where tools are now recorded as registered only on the local host were `pbench-register-tool` and `pbench-register-tool-set` are invoked.  As a result of this change, the following behavioral changes follow:
+
+ * The process of registering tools on local or remote hosts no longer validates that those tools are available during tool registration
+ * All tools registered prior to installing `v0.70.0-beta` must be re-registered; tools registered locally or remotely on a controller with a version of the `pbench-agent` prior to `v0.70.0` will be ignored until they are re-registered
+
+For both the Pbench Agent and the Server we have removed the use of the Software Collections Library (SCL) in order to provide a Python 3 environment for RHEL 7 & CentOS 7 distributions.  We now rely on the Python 3 provided with RHEL 7.7 and later.
+
+We did not bump the "major" release version number with these changes because we still don't consider all the necessary functionality in place for such a major version bump.
+
+Installation
+====
+There are ansible playbooks to install the `pbench-agent` and the pieces needed (key and config files) to be able to send results to a server.
+
+There are no other installation changes in this release: see the [Getting Started Guide](https://distributed-system-analysis.github.io/pbench/start.html) for how to install or update.
+
+After installation or update, you should have version `0.70.0-1gXXXXXXXXX` of the `pbench-agent` RPM installed.
```

Comment:
I'll update the `XXXXXXXX` with the actual commit ID once we finalize the release.